services/worker/src/worker/job_runners/config/parquet_and_info.py

```diff
@@ -1064,6 +1214,7 @@ def compute_config_parquet_and_info_response(
         for repo_file in target_dataset_info.siblings
         if repo_file.rfilename.startswith(f"{config}/") and repo_file.rfilename.endswith(".parquet")
     ]
+    repo_files.sort(key=lambda repo_file: repo_file.rfilename)  # type: ignore
```

Comment:
indeed, lambda functions and mypy are enemies. https://stackoverflow.com/questions/33833881/is-it-possible-to-type-hint-a-lambda-function

alternative: create a function and call it by its name (ie: no lambda)

Reply:
what is the error?

Reply:
oh, ok, so it's because huggingface_hub is not typed? (clearer alternative could be to raise if it's not an instance of str)

Reply:
yea a ValueError makes more sense - I'll update it